CHANGELOG.md

```diff
@@ -103,6 +103,16 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 [3906]: https://github.com/emissary-ingress/emissary/issues/3906
 
+## [2.1.2] December 16, 2021
```

Comment:
This, of course, would need to be updated to reflect the release date (current target is January 20, 2022).

Reply:
This and the following date are done by `make generate` and keep overwriting my `TBD` entry for the date (I'm assuming because there is already a `2,2` version above listed as `TBD` :cry: